cmd/bucket-replication.go

```diff
@@ -2283,3 +2284,77 @@ func saveResyncStatus(ctx context.Context, bucket string, brs BucketReplicationR
 	configFile := path.Join(bucketMetaPrefix, bucket, replicationDir, resyncFileName)
 	return saveConfig(ctx, objectAPI, configFile, buf)
 }
+
+// getReplicationDiff returns unreplicated objects in a channel
+func getReplicationDiff(ctx context.Context, objAPI ObjectLayer, bucket string, diffCh chan interface{}, opts repl.DiffOpts) error {
```

Comment:
Any reason this cannot be typed?
```suggestion
func getReplicationDiff(ctx context.Context, objAPI ObjectLayer, bucket string, diffCh chan<- repl.DiffInfo, opts repl.DiffOpts) error {
```

Reply:
Fixed.